### PR TITLE
Fix typos in content

### PR DESCRIPTION
**Pull request by Nikomus**
Misc spelling fixes while working on other things. May have implications for paths/triggers working correctly.

@TheLabCollective/developers

### CODE REVIEW

File: content/messages/pigs-keep.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ author: AL
 triggered_by:
   - init|start|pigs|keep
 ---
-Thank you, I hope the rest of the annimal welfare team will be ok with this....
\ No newline at end of file
+
+Thank you, I hope the rest of the animal welfare team will be ok with this....
```
Comment (katjam) [inline content/messages/pigs-keep.md:8]: What changed?

**Review by katjam — APPROVED** [review]
Looks good. Thanks for thinking of this.

File: content/messages/pigs-keep.md (hunk 1 above)
Comment (Nikomus) [reply content/messages/pigs-keep.md:8]: It was "annimal"

Comment (katjam) [reply content/messages/pigs-keep.md:8]: I tried so hard to see that.

**Pull request merged**